src/test/java/com/wikia/webdriver/common/templates/fandom/AdsFandomTestTemplate.java

```diff
@@ -0,0 +1,9 @@
+package com.wikia.webdriver.common.templates.fandom;
+
+public class AdsFandomTestTemplate extends FandomTestTemplate {
+
+  @Override
+  protected void loadFirstPage() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1186)